features/support/env.rb

```diff
@@ -75,7 +75,7 @@
 Capybara.register_driver :poltergeist do |app|
   options = {
       :port => (51674 + ENV['TEST_ENV_NUMBER'].to_i),
-      :js_errors => true,
+      :js_errors => false,
```

Comment:
A javascript error is thrown on this cucumber without this, possibly due to the phantom JS issues